R/pkg/R/DataFrame.R

```diff
@@ -1231,6 +1296,24 @@ setMethod("unionAll",
             dataFrame(unioned)
           })
 
+setGeneric("rbind", signature = "...")
+
+rbind.SparkDataFrames <- function(x, ..., deparse.level = 1) {
+  allargs <- list(...)
+  if (nargs() == 3) {
+    unionAll(x, ...)
+  } else {
+    unionAll(x, Recall(..., deparse.level = 1))
+  }
+}
+
+#' @rdname rbind
+#' @aliases unionAll
+setMethod("rbind",
+          signature(... = "DataFrame"),
+          rbind.SparkDataFrames
```

Comment:
Its a good feature to have but I think its also fine to just support the two argument case if the other one adds too much complexity